README.md

```diff
@@ -7,62 +7,28 @@
-    <img alt="GitHub forks" src="https://img.shields.io/github/forks/ACM-VIT/dolos?style=for-the-badge"/>
-    <img alt="GitHub stars" src="https://img.shields.io/github/stars/ACM-VIT/dolos?style=for-the-badge"/>
-    <img alt="Github Actions" src="https://github.com/ACM-VIT/dolos/workflows/Node.js%20CI/badge.svg" />
-</p>
-<img alt="GitHub Hacktoberfest combined status" src="https://img.shields.io/github/hacktoberfest/2020/ACM-VIT/dolos?style=for-the-badge">
+[![made-by-acm](https://img.shields.io/badge/MADE%20BY-ACM%20VIT-blue?style=for-the-badge)](https://acmvit.in/)
+![GitHub last commit](https://img.shields.io/github/last-commit/ACM-VIT/dolos?style=for-the-badge)
+![GitHub repo size](https://img.shields.io/github/repo-size/ACM-VIT/dolos?style=for-the-badge)
+[![License](https://img.shields.io/badge/License-MIT-green.svg?style=for-the-badge)](https://en.wikipedia.org/wiki/MIT_License)
+[![GitHub forks](https://img.shields.io/github/forks/ACM-VIT/dolos?style=for-the-badge)](https://github.com/ACM-VIT/dolos/network)
+[![GitHub stars](https://img.shields.io/github/stars/ACM-VIT/dolos?style=for-the-badge)](https://github.com/ACM-VIT/dolos/stargazers)
+![Github Actions](https://img.shields.io/github/workflow/status/ACM-VIT/dolos/Node.js%20CI?style=for-the-badge)
+![GitHub Hacktoberfest combined status](https://img.shields.io/github/hacktoberfest/2020/ACM-VIT/dolos?style=for-the-badge)
 
+</div>
 
 ---
 
 Ever wanted to get random data for that app you were building, say no more. Dolos, the god of deception and trickery is here for you. 
 
-
 ---
-## Contribute
-To take part in the development of this project, refer the github projects board. We have setup beginner level issues, and labelled them as well ! Feel free to claim any issue (or open a new one) and contribute to the development of the project.
-
 
----
-## Submitting a Pull Request
-
- * Fork the repository by clicking the fork button on top right corner of the page
- * Clone the target repository. To clone, click on the clone button and copy the https address. Then run 
- <pre><code>git clone [HTTPS-ADDRESS]</code></pre>
-* Go to the cloned directory by running 
-<pre><code>cd [NAME-OF-REPO]</code></pre>
-* Create a new branch. Use 
-<pre><code> git checkout -b [YOUR-BRANCH-NAME]</code></pre>
-* Make your changes to the code. Add changes to your branch by using 
-<pre><code>git add .</code></pre>
-* Commit the chanes by executing
-<pre><code>git commit -m "your msg"</code></pre>
-* Push to remote. To do this, run 
-<pre><code>git push origin [YOUR-BRANCH-NAME]</code></pre>
-* Create a pull request. Go to the target repository and click on the "Compare & pull request" button. **Make sure your PR description mentions which issues you're solving.**
-<img src="https://drive.google.com/u/1/uc?id=1f9JKAR-kRvCRGxIs_SAvegaYDPx53T9G&export=download"></img>
-* Wait for your request to be accepted. 
-
----
-## Guidelines for Pull Request
-
-<!-- general guidelines here -->
```

Comment:
all this content needs to be in a file right ? it's moving content, not deleting it.

Reply:
you are right, fixing it right now

Reply:
b60ef68 added missing content